src/pytest_mock_resources/fixture/database/relational/postgresql.py

```diff
@@ -4,6 +4,7 @@
 from pytest_mock_resources.container.postgres import get_sqlalchemy_engine, PostgresConfig
 from pytest_mock_resources.fixture.database.generic import assign_fixture_credentials
 from pytest_mock_resources.fixture.database.relational.generic import EngineManager
+from sqlalchemy.ext.asyncio import AsyncSession
```

Comment:
this will break backwards compatibility with sqlalchemy <1.4. This import should probably happen from within that function, `try/except ImportError`'d, with a message mentioning the version requirement